### PR TITLE
Fixes issues with watchOS target

### PR DESCRIPTION
**Pull request by simonmitchell**
Fixes docs
Adds version check to earliestBeginDate
Fixes watch. bridging header

### CODE REVIEW

File: ThunderRequest/TSCRequestController.m [hunk 1]
```diff
@@ -608,7 +608,7 @@ - (void)scheduleDownloadRequest:(TSCRequest *)request on:(NSDate *)beginDate pro
 			NSURLRequest *normalisedRequest = [self backgroundableRequestObjectFromTSCRequest:request];
 			NSURLSessionDownloadTask *task = [welf.backgroundSession downloadTaskWithRequest:normalisedRequest];
             
-            if (@available(iOS 11.0, *)) {
+            if (@available(iOS 11.0, *) && @available(watchOS 4.0, *)) {
```
Comment (ryanbourneuk) [inline ThunderRequest/TSCRequestController.m]: Should this be `and`, or, `or`?

Comment (MattCheetham) [inline ThunderRequest/TSCRequestController.m]: You should be able to make this `if @available(iOS 11.0, watchOS 4.0, *) {`

**Review by MattCheetham — CHANGES_REQUESTED** [review]

File: ThunderRequest/TSCRequestController.m (hunk 1 above)
Comment (simonmitchell) [reply ThunderRequest/TSCRequestController.m]: Neither of us were right 😆 turns out you can put multiple os' in the @available parameters

Comment (simonmitchell) [reply ThunderRequest/TSCRequestController.m]: Just came to the same conclusion! ❤️

**Review by ryanbourneuk — APPROVED** [review]

**Review by MattCheetham — APPROVED** [review]

**Pull request merged**